internal/aws/cloud.go

```diff
@@ -77,6 +81,7 @@ func New(cfg CloudConfig, clusterName string, mc metric.Collector) (CloudAPI, er
 		ec2.New(awsSession, regionCfg),
 		elbv2.New(awsSession, regionCfg),
 		iam.New(awsSession, regionCfg),
+		shield.New(awsSession, &aws.Config{Region: aws.String("us-east-1")}),
```

Comment:
Any reason this should be hard-coded to `"us-east-1"` and not just be `regionCfg`?

Reply:
Ah, thanks! I was looking for that link in the documentation, since I saw that the service itself was available in a lot more regions.